FileSystem/Workbook.js

```diff
@@ -24,3 +41,73 @@ function openWorkbook(path) {
   openURL(file.getUrl(), file.getName() + ' file open in new tab');
   return file;
 }
+
+/**
+ * Prompts the user to provide the local file path for the current active spreadsheet.
```

Comment:
Update comments : In what scenario does it prompt the user?

Reply:
Fixed, wrote the 2 main scenarios